src/valhalla_export_edges.cc

```diff
@@ -223,15 +232,18 @@ int main(int argc, char *argv[]) {
          edge.e->trans_down() || edge.e->IsTransitLine()) //these 2 should never happen
         continue;
 
-      //get the opposing edge as well
-      edge_t opposing_edge = opposing(reader, tile, edge);
-      edge_set.set(tile_set.find(opposing_edge.i.Tile_Base())->second + opposing_edge.i.id());
-      ++set;
-
       //shortcuts arent real and maybe we dont want ferries
       if(edge.e->is_shortcut() || (!ferries && edge.e->use() == Use::kFerry))
         continue;
 
+      //get the opposing edge as well (ensure a valid edge is returned)
```

Comment:
ah ok. well if thats the case then it may as well be moved all the way down passed the last continue where it checks for names or not. i just want to make sure that nothing gets double counted haha! and i think that would be the case regardless of where this goes right?